bar-trender-front-end/src/components/Navbars/MainNavbar.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import barTrender from '../../assets/img/barTrender60.png';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123646)
```suggestion
import barTrender from "../../assets/img/barTrender60.png";
```